lib/post_creator.rb

```diff
@@ -159,6 +159,19 @@ def valid?
         return false
       end
 
+      if @topic&.slow_mode_seconds.to_i > 0
+        tu = TopicUser.find_by(user: @user, topic: @topic)
+
+        if tu&.last_posted_at
+          threshold = tu.last_posted_at + @topic.slow_mode_seconds.seconds
+
+          if DateTime.now < threshold
```

Comment:
```suggestion
          if Time.zone.now < threshold
```
We prefer this in our codebase.